src/components/Confirm/PdfPreview.js

```diff
@@ -0,0 +1,67 @@
+import { h, Component } from 'preact'
+import PDFObject from 'pdfobject'
+import {preventDefaultOnClick} from '../utils'
+
+const FallbackView = ({blob}) => {
+  const downloadPDF = (e) => {
+    window.navigator.msSaveOrOpenBlob(blob, 'document.pdf')
+  }
+  return (
+    <div>
+      <a href='#' onClick={preventDefaultOnClick(downloadPDF)}><div id='pdfIcon'></div></a>
```

Comment:
Can't this be `<a href onClick className={styles.pdfIcon} />`?

Seems silly to have a `<div id='pdfIcon' />` (a singleton inner DOM element) - I can't see that it does anything?

You can just

```
.pdfIcon {
  display: inline-block;
  //...
}
```

in your SCSS